plugins/out_s3/s3.c

```diff
@@ -1144,6 +1191,8 @@ static int put_all_chunks(struct flb_s3 *ctx)
     struct mk_list *f_head;
     struct flb_fstore_file *fsf;
     struct flb_fstore_stream *fs_stream;
+    void *payload_buf = NULL;
+    size_t payload_size = 0;
```

Comment:
where do you free the `payload_buf` here?

Reply:
May I know which part are you referring to?